CMakeLists.txt

```diff
@@ -98,7 +98,7 @@ pkg_check_modules(LZ4 REQUIRED IMPORTED_TARGET liblz4)
 
 # cURL
 set(curl_target "")
-if (ENABLE_HTTP OR ENABLE_DATA_TOOLS)
+if (ENABLE_HTTP)
```

Comment:
actually we don't need curl for data tools. there's only `valhalla_add_elevation.cc` which uses the curler, but even then it's protected with `ENABLE_HTTP`:

https://github.com/valhalla/valhalla/blob/babac1f960097007d1bcdba82d63346e793a4480/src/baldr/curler.cc#L8